source/jquery.flot.js

```diff
@@ -984,6 +987,25 @@ Licensed under the MIT license.
             executeHooks(hooks.bindEvents, [eventHolder]);
         }
 
+        function addEventHandler(event, handler, eventHolder, priority) {
+            var key = eventHolder + event;
+            var eventlist = eventManager[key];
+            if (eventlist === undefined) {
+                eventlist = [];
+            }
+
+            // var newEventList = [];
+            eventlist.push({"event": event, "handler": handler, "eventHolder": eventHolder, "priority": priority});
+            eventlist.sort(function(a, b) { return b.priority - a.priority });
```

Comment:
Would it be possible to _insert_ the new event at the proper place, instead of doing “push to end, then global sort to restore order” ?

Reply:
No, because the ordering of the events firing is completely based off of the order in which the handlers registered through the 'bind' operation.

Reply:
To be clear, I'm talking about lines 998 and 999 (i.e. this suggestion is independent of the bind operations that happen in the loop below), so I don’t understand what "the ordering of the events firing" has to do with how we maintain the order of the event list.